WooCommerce/Classes/ViewRelated/Orders/Order Creation/PaymentSection/FeeLineDetails.swift

```diff
@@ -0,0 +1,148 @@
+                        .background(Color(.listForeground))
+                        .fixedSize(horizontal: false, vertical: true)
+                        .onTapGesture {
+                            focusAmountInput = true
+                        }
+                        .padding(.horizontal, insets: safeAreaInsets)
+                    }
+                    .background(Color(.listForeground))
+                    .addingTopAndBottomDividers()
+
+                    Spacer(minLength: Layout.sectionSpacing)
+
+                    if viewModel.isExistingFeeLine {
+                        Section {
+                            Button(Localization.remove) {
+                                viewModel.didSelectSave(nil)
+                                presentation.wrappedValue.dismiss()
+                            }
+                            .padding()
+                            .frame(maxWidth: .infinity, alignment: .center)
+                            .foregroundColor(Color(.error))
+                            .padding(.horizontal, insets: safeAreaInsets)
+                            .addingTopAndBottomDividers()
+                        }
+                        .background(Color(.listForeground))
+                    }
+                }
+            }
+            .background(Color(.listBackground))
+            .ignoresSafeArea(.container, edges: [.horizontal, .bottom])
+            .navigationTitle(viewModel.isExistingFeeLine ? Localization.fee : Localization.addFee)
+            .navigationBarTitleDisplayMode(.inline)
+            .toolbar {
+                ToolbarItem(placement: .cancellationAction) {
+                    Button(Localization.close) {
+                        presentation.wrappedValue.dismiss()
+                    }
+                }
+                ToolbarItem(placement: .primaryAction) {
+                    Button(Localization.done) {
+                        viewModel.saveData()
+                        presentation.wrappedValue.dismiss()
+                    }
+                    .disabled(viewModel.shouldDisableDoneButton)
+                }
+            }
+        }
+        .wooNavigationBarStyle()
+    }
+
+    private var inputFixedField: some View {
+        TitleAndTextFieldRow(title: String.localizedStringWithFormat(Localization.amountField, viewModel.storeCurrencySymbol),
+                             placeholder: "",
+                             text: .constant(viewModel.formattedAmount),
+                             symbol: nil,
+                             keyboardType: .decimalPad)
+            .foregroundColor(Color(viewModel.amountTextColor))
+            .disabled(true)
+    }
+
+    private var inputPercentageField: some View {
+        TitleAndTextFieldRow(title: String.localizedStringWithFormat(Localization.amountField, viewModel.percentSymbol),
+                             placeholder: "",
+                             text: .constant(viewModel.amount),
```

Comment:
I noticed that showing this text means the percentage amount field doesn't display a placeholder. That's probably ok for now if we're going to get rid of the fake text field and use the `BindableTextfield` directly in this view, but otherwise we should probably make sure a placeholder is shown.

Fixed amount (with placeholder)|Percentage amount (no placeholder)
-|-
![Simulator Screen Shot - iPhone 13 Pro - 2022-02-22 at 09 32 38](https://user-images.githubusercontent.com/8658164/155111274-177c57dc-bff8-4d3b-be3a-15ae3945edb3.png)|![Simulator Screen Shot - iPhone 13 Pro - 2022-02-22 at 09 32 33](https://user-images.githubusercontent.com/8658164/155111284-2058f4d5-10b3-4a11-8918-c9ee7b426201.png)

Reply:
Agree! Added a placeholder in 621a89c so I won't forget it after applying `BindableTextfield` refactor.